### PR TITLE
Fix Kafka tutorial - change remaining topic names to mytopic

### PR DESCRIPTION
**Pull request by orishavit**
### Description

Some parts of the tutorial used the wrong Kafka topic name (probably was changed at some point), which caused the tutorial to not work cleanly. This PR changes all references to the topic name to `mytopic`.

### CODE REVIEW

**Comment by orishavit** [comment]
recheck

**Comment by orishavit** [comment]
I have read and understood the CLA and hereby agree to its terms by making this Pull Request Comment.

**Comment by orishavit** [comment]
recheck

**Comment by orishoshan** [comment]
@orishavit So it turns out this was intentional -- the `transactions` topic is supposed to be used only in the KafkaServerConfig, and elsewhere you're supposed to work with `mytopic`. The problem you experienced was actually caused by a bug, resolved in PR https://github.com/otterize/intents-operator/pull/183.

But this tutorial is pretty confusing, and we'll be reworking it. The other text changes you made, except for those in the KafkaServerConfig - are appropriate corrections. So I suggested a change for the KafkaServerConfig back to `transactions`, leaving the rest as-is.

File: docs/quick-tutorials/k8s-kafka-mtls.mdx [hunk 1]
```diff
@@ -111,15 +111,15 @@ kubectl apply -f https://docs.otterize.com/code-examples/kafka-mtls/kafkaserverc
...
+Upon applying the KafkaServerConfig, an ACL will configure Kafka to allow only authenticated access to the *mytopic* topic by denying all anonymous access.
 This will be the base state, from which we will gradually roll out secure access to Kafka.
 
 ```bash
 kubectl logs -n kafka statefulset/kafka | grep "Processing Acl change" | grep ANONYMOUS | tail -n 1
 ```
 You should see the following output:
 ```
-[2023-05-18 11:49:14,230] INFO Processing Acl change notification for ResourcePattern(resourceType=TOPIC, name=transactions, patternType=LITERAL), versionedAcls : Set(User:ANONYMOUS has DENY permission for operations: ALL from hosts: *, User:* has ALLOW permission for operations: ALL from hosts: *), zkVersion : 0 (kafka.security.authorizer.AclAuthorizer)
+[2023-05-18 11:49:14,230] INFO Processing Acl change notification for ResourcePattern(resourceType=TOPIC, name=mytopic, patternType=LITERAL), versionedAcls : Set(User:ANONYMOUS has DENY permission for operations: ALL from hosts: *, User:* has ALLOW permission for operations: ALL from hosts: *), zkVersion : 0 (kafka.security.authorizer.AclAuthorizer)
```
Comment (orishoshan) [inline docs/quick-tutorials/k8s-kafka-mtls.mdx:122]:
```suggestion
[2023-05-18 11:49:14,230] INFO Processing Acl change notification for ResourcePattern(resourceType=TOPIC, name=transactions, patternType=LITERAL), versionedAcls : Set(User:ANONYMOUS has DENY permission for operations: ALL from hosts: *, User:* has ALLOW permission for operations: ALL from hosts: *), zkVersion : 0 (kafka.security.authorizer.AclAuthorizer)
```

File: static/code-examples/kafka-mtls/kafkaserverconfig.yaml [hunk 2]
```diff
@@ -12,7 +12,7 @@ spec:
     keyFile: /etc/otterize-spire/key.pem
     rootCAFile: /etc/otterize-spire/ca.pem
   topics:
-    - topic: "transactions"
+    - topic: "mytopic"
```
Comment (orishoshan) [inline static/code-examples/kafka-mtls/kafkaserverconfig.yaml:15]:
```suggestion
    - topic: "transactions"
```

**Review by orishoshan — APPROVED** [review]

**Pull request closed without merging**